modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -3,26 +3,23 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use case one includes all of the instructions for setting up an identity provider for your portal when no other identity provider is not configured in Anypoint. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint, you don’t need to map the identity provider for the portal to the identity provider in Anypoint since the default identity provider can be used for the portal. This enables every Salesforce user to have an identity in Anypoint.
+Use case one includes all of the instructions for setting up an identity provider for your portal when no other identity provider is not configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider is used to map the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint.
```

Comment:
```suggestion
Use case one includes all of the instructions for setting up an identity provider for your portal when no other identity provider is configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider is used to map the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint.
```

Reply:
I find this text confusing:

"Since there is no identity provider configured in Anypoint Platform, the default identity provider is used to map the identity provider for the portal to the identity provider in Anypoint Platform."

Is the default identity provider configured in Salesforce? Where does this come from? Is it the AEH default identity provider? If so, it would help if you say this.

Also, I would remove "Use Case 1" from the topic title to shorten it. I would also be explicit about Anypoint Platform in the topic title:

Configuring Single Sign-On When No Identity Provider Is Configured in Anypoint Platform

Reply:
Yes, it is the AEH default IdP. It is also explained in the enabling-sso-for-your-portal.adoc. I'll specify that.
For clarity, I added that it automatically maps the IdP for the portal to the IdP in Anypoint.